packages/esm-patient-banner-app/src/banner/patient-banner.component.tsx

```diff
@@ -42,6 +42,9 @@ const PatientBanner: React.FC<PatientBannerProps> = ({ patient, patientUuid, hid
   }, []);
 
   const isDeceased = Boolean(patient?.deceasedDateTime);
+  // render details button below patient details for workspaces
+  // 520px is the maximum width a workspace occupies
+  const renderDetailsButtonBelow = patientBannerRef.current?.scrollWidth <= 520;
```

Comment:
```suggestion
	const maxWorkspaceWidth = 520;
	
  const showDetailsButtonBelowHeader = !isTabletViewport && (!Boolean(patientBannerRef.current) || patientBannerRef.current.scrollWidth <= maxWorkspaceWidth);
```

The current logic doesn't account for two of the situations Ian mentioned above.

Reply:
Yes @denniskigen. Thanks for pointing this out. I tested it with the condition provided, but the appearance on the desktop view was not as expected. 

![image](https://github.com/openmrs/openmrs-esm-patient-chart/assets/53287480/cedc5b89-8911-4b1e-8ef4-dfd5ac443e36)

Therefore, I decided to only check the width of the banner instead. Whenever the width is less than 520px, the button will be displayed below the header. This way, the button will never appear under the header on tablets, as they are greater than 520px in width.

cc @ibacher